docs/user/security/api-keys/index.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[api-keys]]
+=== API Keys
+
+An API key enables you to create secondary credentials with the same or lower 
+access rights so that you can send requests on behalf of the user.  For example, 
+you might have access to an Elasticsearch cluster and want to create a cron 
+job to extract data from it on a daily basis. You can create an API key tied 
+to your credentials and configure it with the minimum access required by the 
+script, and then put the API key credentials into the cron job.  
+
+You can also use API keys to automate ingestion of new data from 
+remote sources, without a live user interaction. 
+
+You can create API Keys in the {kib} Console and view and invalidate 
+them in *Management > Security > API Keys*.
+
+[role="screenshot"]
+image:user/security/api-keys/images/api-keys.png["API Keys UI"]
+
+[float]
+[[api-keys-security-privileges]]
+=== Security privileges
+
+You must have the `manage_security`, `manage_api_key`, or the `manage_own_api_key` 
+cluster privileges to use API Keys in {kib}. You can manage roles in 
+*Management > Security > Roles*, or use the <<role-management-api, {kib} Role Management API>>. 
+
+[float]
+[[api-keys-service]]
+=== {es} API key service
+
+The {es} API key service is automatically enabled when you configure 
+{ref}/configuring-tls.html#tls-http[TLS on the HTTP interface]. 
+This ensures that API keys are not sent in clear-text by clients.
+
+If HTTPS connections are not enabled between {kib} and {es}, you will 
```

Comment:
How about:

When HTTPS connections are not enabled between Kibana and Elasticsearch, you cannot use or manage API keys, and you get an error message.